src/patternfly/components/BackgroundImage/backgroundImage.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="pf-c-background-image">
```

Comment:
Modifier or no modifier?

Reply:
oh yes, that should be in there. I took it out when was trying to debug something :(